app/models/document.rb

```diff
@@ -0,0 +1,7 @@
+require 'laa_crime_schemas'
+
+class Document < LaaCrimeSchemas::Structs::Document
+  def file_type
```

Comment:
If we think this may be useful, we can extract this helper method to the gem Structs so both Apply/Review can use it. But no urgency for now.

Reply:
Yeah we considered that but thought it would not be required on apply. Depends on if apply will ever present/need the file extension...

Reply:
I'm not even sure caseworkers need it if it's visible in the file name but 🤷‍♀️